Actions/.Modules/ReadSettings.psm1

```diff
@@ -549,6 +549,64 @@ function ReadSettings {
         $settings.projectName = $project # Default to project path as project name
     }
 
+    $ghEnvironments = @(GetGitHubEnvironments)
```

Comment:
This will be called on every `ReadSettings` call, right? Given that `GetGitHubEnvironments` makes an HTTP call, it seems excessive and unnecessary.